manuscript/develop/devspace-helm.md

```diff
@@ -25,12 +25,12 @@ devspace dev --namespace dev
 
 go run .
 
-# Check the newly opened browser tab
+# Open localhost:8080 in a browser
```

Comment:
It should open a new browser tab automatically. Was that not the case?

Reply:
I didn't mean to change that, sorry.  Changing it back